src/utils.ts

```diff
@@ -147,26 +148,15 @@ const prepareData = (
   return events.filter((e) => hashValuesFromKeys(e, keysToHash));
 };
 
-const shouldCachePayload = (
-  url: string,
-  baseUrl: string,
-  config: ConfigType
-) => {
+const shouldCachePayload = (url: string, baseUrl: string) => {
   const requestUrl = new URL(url);
   const baseOriginUrl = new URL(baseUrl);
 
   // Origin is needed for 'localhost' testing rather than hostname
-  if (requestUrl.origin == baseOriginUrl.origin) return false;
-  if (config.allowedDomains?.length) {
-    return config.allowedDomains.some((domain) => {
-      return requestUrl.hostname.includes(domain);
-    });
-  }
-  if (config.ignoredDomains?.length) {
-    return !config.ignoredDomains.some((domain) => {
-      return requestUrl.hostname.includes(domain);
-    });
```

Comment:
I have removed this check since we are ignoring it on the interceptors level

Reply:
👍